### DISCUSSION TITLE
Order of conditional variables changes outcome partial RDA

### DISCUSSION THREAD

**Discussion by britteheijink** [vegandevs/vegan#590] [Q&A] [answered]
Hi all,

I'm running a partial RDA with three conditioning variables. I noticed that the resulting RDA object is different based on the order of the conditioning variabales (i.e. eigenvalues and inertia values are different). Is it not possible to run a partial RDA with multiple conditional variables or am I doing something wrong/is this expected? 

A simplified example of my code is:

`
rda(x ~a + b +c,  Condition(d + e + f), data = z, scale = TRUE)
`

I'm working in vegan 2.6.4 and R version 4.2.2

Thank you for your help!

**Comment by jarioksa** [comment, marked as answer]
It won’t. We need a reproducible example. BTW, your syntax is wrong. It should be `y ~ x + Condition(z), data =…` (NB the `+` sign).

**Reply by britteheijink** [reply]
I did use the `+` sign in my actual code, but found another syntax error in the actual code. I used `Condition (d, e ,f)` vs `Condition (d +e +f)`. Problem solved. Thank you for your time and sorry to bother you for this kind of mistake!
